docs/quick-start.md

```diff
@@ -1,19 +1,20 @@
 # Quick Start
+In this example we will send 1 _AE_ coin from one account to another
 
 ## 1. Specify imports
 For the following snippets in the guide you need to specify multiple imports. Most imports like `Universal`, `MemoryAccount` & `Node` are [Stamps](https://stampit.js.org/essentials/what-is-a-stamp) that compose certain functionalities. Others like `AmountFormatter` & `Crypto` are util modules with typical function exports.
 
 ```js
-import {
+const {
   Universal,
   MemoryAccount,
   Node,
   AmountFormatter,
   Crypto
-} from '@aeternity/aepp-sdk'
+} = require('@aeternity/aepp-sdk')
 ```
 
-## 2. Create a Keypair
+## 2. Create a Keypair for sender
```

Comment:
I wanted to explain there is a Sender and Recipient.
`<RECIPIENT_PUBLIC_KEY>` was set before, but there was no note about Sender